### PR TITLE
make iframes responsive on patient view page

### PR DESCRIPTION
**Pull request by dippindots**
make iframes on patient view page responsive.
fix https://github.com/cBioPortal/cbioportal/issues/5692

### CODE REVIEW

**Comment by dippindots** [comment]
@alisman I found that in our WindowStore, the window size is the window's inner size. But sometimes the content in the window will larger than the window, so I added the scroll size into it. But Maybe it is better to create a DocumentStore to?

File: src/pages/patientView/pathologyReport/PathologyReport.tsx [hunk 1]
```diff
@@ -35,26 +36,25 @@ export default class PathologyReport extends React.Component<IPathologyReportPro
...
+    // }
 
     handleSelection(){
         this.setState({ pdfUrl:this.buildPDFUrl(this.pdfSelectList.options[this.pdfSelectList.selectedIndex].value) });
     }
 
     render(){
 
-        return (<div>
+        return (<div style={{position: "relative"}}>
```
Comment (alisman) [inline src/pages/patientView/pathologyReport/PathologyReport.tsx]: what's reason for position:relative?

Comment (dippindots) [reply src/pages/patientView/pathologyReport/PathologyReport.tsx]: hmm, there is no special reason for that. I can clean the unneeded style.

**Comment by dippindots** [comment]
Here are two examples URL:
1. https://cbioportal.mskcc.org/patient/pathologyReport?caseId=TCGA-OR-A5J1&studyId=acc_tcga_pan_can_atlas_2018
2. https://cbioportal.mskcc.org/patient?studyId=mskimpact&caseId=P-0022312

**Comment by jjgao** [comment]
Thanks, @dippindots .

I think it's probably good idea to remove the footer now? @alisman

**Comment by dippindots** [comment]
@jjgao @alisman Sorry I forgot to ask. Do we want to hide the footer for all tabs in patient view page or just for these three tabs?

**Comment by jjgao** [comment]
@dippindots I think we should remove it from patient view if @alisman agrees.

File: src/shared/components/window/WindowStore.ts [hunk 2]
```diff
@@ -20,13 +21,14 @@ class WindowStore {
     @action
     private setWindowSize() {
         this.size = {
-            width: this.windowObj.innerWidth,
-            height: this.windowObj.innerHeight
+            width: Math.max(this.windowObj.innerWidth, this.documentObj.documentElement.scrollWidth),
```
Comment (alisman) [inline src/shared/components/window/WindowStore.ts]: I'm curious why we do this.  Let me know when you get in.  It does appear to be working well!

**Pull request merged**